docs/specification.md

```diff
@@ -101,7 +101,7 @@ data Config contractMetadata proposalMetadata = Config
   }
 ```
 
-***TODO:*** Contract metadata is not implemented yet, as well as the signature change of `decisionLambda`.
+***TODO [#29]:*** Contract metadata is not implemented yet, as well as the signature change of `decisionLambda`.
```

Comment:
Shouldn't it be #42? At least the metadata part.

Reply:
Ah hm, I meant not FA2 metadata rather the new `contractMetadata` type argument. 

Is it worth using a different name for it to avoid confusion? Though then for consistency we have to rename `proposalMetadata` as well then.

Reply:
Hmm, I'd call it `contractExtra` then (for example). `proposalMetadata` sounds less confusing because it's clear that it applies to a proposal (something small within the whole contract) while TZIP-16 is about metadata of a contract.

Reply:
I mean, since `proposalMetadata` is already used in many places and is not very confusing, I think we can keep it.

Reply:
Okay, I think we better do the replacement.